Library/String+Currency.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import UIKit.UIColor
+import UIKit.UIFont
+
+public func attributedCurrencyString(
+  currencySymbol: String,
+  amount: Double,
+  fractionDigits: UInt,
+  font: UIFont,
+  superscriptFont: UIFont,
+  foregroundColor: UIColor
+  ) -> NSAttributedString {
+  // Drop decimal places
+  let formattedString = String(format: "\(currencySymbol)%.\(fractionDigits)f", amount)
```

Comment:
We strip down unnecessary decimal points..

Reply:
Any reason not to use the formatter we already have? 
```
Format.formattedCurrency(_ amount: Any, country: Project.Country, omitCurrencyCode: Bool = true, env: Environment = AppEnvironment.current)
``` 
... returns a formatted currency amount which is localized and uses a cached `NumberFormatter`.

Reply:
Ah, this is great...I'll try using the formatter...had no idea we already have it!